xbmc/platform/darwin/osx/SDLMain.mm

```diff
@@ -442,8 +442,8 @@ - (void) deviceDidUnMountNotification:(NSNotification *) note
 #define VK_MEDIA_PREV_TRACK 0xB1
 #define VK_MEDIA_STOP       0xB2
 #define VK_MEDIA_PLAY_PAUSE 0xB3
-#define VK_REWIND           0x9D
-#define VK_FAST_FWD         0x9E
+#define VK_REWIND           0xB1
```

Comment:
Should I remove `VK_REWIND` and `VK_FAST_FWD` and replace them with `VK_MEDIA_PREV_TRACK` and `VK_MEDIA_NEXT_TRACK`?